modules/npmscripts.config.js

```diff
@@ -26,6 +26,144 @@ const CHECK_AND_FIX_GLOBS = [
 
 module.exports = {
 	check: CHECK_AND_FIX_GLOBS,
+	defaults: {
```

Comment:
I think I'll have some thoughts on the naming and inheritance/override model here, but those probably belong better on [the other PR](https://github.com/liferay/liferay-frontend-projects/pull/420), so I'll go and take a look at that one now.